[2024-07-19T07:35:09.923+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: crawling_dag_v2.read_s3_and_compare_links manual__2024-07-19T07:30:05.226416+00:00 [queued]>
[2024-07-19T07:35:09.934+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: crawling_dag_v2.read_s3_and_compare_links manual__2024-07-19T07:30:05.226416+00:00 [queued]>
[2024-07-19T07:35:09.935+0000] {taskinstance.py:2193} INFO - Starting attempt 2 of 2
[2024-07-19T07:35:09.962+0000] {taskinstance.py:2217} INFO - Executing <Task(PythonOperator): read_s3_and_compare_links> on 2024-07-19 07:30:05.226416+00:00
[2024-07-19T07:35:09.970+0000] {standard_task_runner.py:60} INFO - Started process 388 to run task
[2024-07-19T07:35:09.976+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'crawling_dag_v2', 'read_s3_and_compare_links', 'manual__2024-07-19T07:30:05.226416+00:00', '--job-id', '451', '--raw', '--subdir', 'DAGS_FOLDER/musinsa_dag.py', '--cfg-path', '/tmp/tmp46mauk4k']
[2024-07-19T07:35:09.980+0000] {standard_task_runner.py:88} INFO - Job 451: Subtask read_s3_and_compare_links
[2024-07-19T07:35:10.046+0000] {task_command.py:423} INFO - Running <TaskInstance: crawling_dag_v2.read_s3_and_compare_links manual__2024-07-19T07:30:05.226416+00:00 [running]> on host 9a455ef6c02c
[2024-07-19T07:35:10.161+0000] {taskinstance.py:2513} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='crawling_dag_v2' AIRFLOW_CTX_TASK_ID='read_s3_and_compare_links' AIRFLOW_CTX_EXECUTION_DATE='2024-07-19T07:30:05.226416+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-07-19T07:30:05.226416+00:00'
[2024-07-19T07:35:10.174+0000] {base.py:83} INFO - Using connection ID 'aws_s3' for task execution.
[2024-07-19T07:35:10.175+0000] {connection_wrapper.py:384} INFO - AWS Connection (conn_id='aws_s3', conn_type='aws') credentials retrieved from login and password.
[2024-07-19T07:35:46.755+0000] {logging_mixin.py:188} INFO - Current number of unique href links: 43
[2024-07-19T07:35:50.415+0000] {local_task_job_runner.py:302} WARNING - State of this instance has been externally set to failed. Terminating instance.
[2024-07-19T07:35:50.422+0000] {process_utils.py:131} INFO - Sending 15 to group 388. PIDs of all processes in the group: [388]
[2024-07-19T07:35:50.423+0000] {process_utils.py:86} INFO - Sending the signal 15 to group 388
[2024-07-19T07:35:50.424+0000] {taskinstance.py:2483} ERROR - Received SIGTERM. Terminating subprocesses.
[2024-07-19T07:35:50.523+0000] {process_utils.py:79} INFO - Process psutil.Process(pid=388, status='terminated', exitcode=0, started='07:35:09') (388) terminated with exit code 0
